sys/linux/socket_netlink_route_sched.txt

```diff
@@ -76,11 +90,37 @@ type tca_kind_options_t[NAME, VALUES] {
 	TCA_OPTIONS	nlattr[TCA_OPTIONS, VALUES]
 }
 
+type tca_kind_comp_t[NAME, OPTION, VALUES] {
+	TCA_KIND	nlattr[TCA_KIND, string[NAME]]
+	TCA_OPTIONS	nlattr[TCA_OPTIONS, OPTION]
+	TCA_OPTIONS	nlattr[TCA_OPTIONS, VALUES]
+}
+
 # ------------------------------ tc qdisc ------------------------------
-# TODO: qdisc_kind left "choke", "clsact", "codel", "drr", "dsmark", "bfifo", "pfifo", "pfifo_head_drop", "pfifo_fast", "fq", "fq_codel", "gred", "hfsc", "hhf", "htb", "ingress", "mqprio", "multiq", "netem", "pie", "prio", "qfq", "red", "rr", "sfb", "sfq", "tbf"
 qdisc_kind_options [
-	q_cbq	tca_kind_options_t["cbq", q_cbq_options]
-	q_cbs	tca_kind_options_t["cbs", q_cbs_options]
+	q_cbq			tca_kind_options_t["cbq", q_cbq_options]
```

Comment:
Should we do array[q_cbq_options] here? and similarly update other cases?

Reply:
Yes, we should.